src/peer-state.js

```diff
@@ -0,0 +1,63 @@
+import { getUnixTimestamp } from './utils.js';
+
+const peerStateMap = new Map();
+
+function getPeerState(peer) {
+  return (
+    peerStateMap.get(peer.alias) ?? {
+      incomingHyperdriveKeyHex: null,
+      heartbeatIntervalId: null,
+      incomingHeartbeat: {
+        value: null,
```

Comment:
Isn't this always the outgoing hyperdrive version? The key feels unnecessarily vague.